builder/amazon/common/step_create_tags.go

```diff
@@ -101,6 +101,10 @@ func (s *StepCreateTags) Run(_ context.Context, state multistep.StateBag) multis
 					awsErr.Code() == "InvalidSnapshot.NotFound" {
 					return false, nil
 				}
+			} else if awsErr.Code() == "RequestError" {
```

Comment:
I think the indentation in some of these might be off. I think this should be an else of the inner if (`if awsErr.Code() == "InvalidAMIID.NotFound"...`)